packages/interface/src/components/HelpButton.tsx

```diff
@@ -0,0 +1,44 @@
+import * as DropdownMenu from "@radix-ui/react-dropdown-menu";
+import Image from "next/image";
+import React from "react";
+import { CiCircleQuestion } from "react-icons/ci";
+
+import { IconButton } from "./ui/Button";
+
+const feedbackUrl = process.env.NEXT_PUBLIC_FEEDBACK_URL!;
+
+export const HelpButton = (): JSX.Element => (
+  <DropdownMenu.Root>
+    <DropdownMenu.Trigger asChild>
+      <IconButton className="w-[100px] px-0  text-gray-600" icon={CiCircleQuestion} variant="ghost">
+        Help
+      </IconButton>
+    </DropdownMenu.Trigger>
+
+    <DropdownMenu.Portal>
+      <DropdownMenu.Content className="DropdownMenuContent" sideOffset={25}>
+        <DropdownMenu.Item className="DropdownMenuItem">
+          <a className="block underline" href="/#FAQ">
```

Comment:
I would suggest a `w-full` here, also in below `<a>` tags, since it's an option of a dropdown menu, adding `w-full` make not only the text itself could be clicked, but also the full block.